test/Database/LSMTree/Class/Monoidal.hs

```diff
@@ -58,8 +58,9 @@ class (IsSession (Session h)) => IsTableHandle h where
         -> m (V.Vector (QueryResult k v))
 
     newCursor ::
-           IOLike m
-        => h m k v
+           (IOLike m, SerialiseKey k)
+        => Maybe k
+        -> h m k v
```

Comment:
This reminds me, we should swap the order of arguments in the public API so that the table handle comes first.

No need to change in this PR though, it's consistent with the rest of the code.